.azuredevops/pull_request_templates/branches/production.md

```diff
@@ -0,0 +1,9 @@
+## Before deploying to production, please ensure:
```

Comment:
Consider linking to the [Deployment policy doc](https://github.com/dotnet/core-eng/blob/master/Documentation/Policy/DeploymentPolicy.md), which is the source of truth for these statements. A summary here is good, especially to highlight important actions.

Reply:
That's a good plan; I was basing these checkboxes on that so it might be good to just say "review this."